CHANGELOG.md

```diff
@@ -26,6 +26,12 @@ INTERNAL NOTE
 -->
 ## Flutter 3.27 Changes
 
+### [3.27.1](https://github.com/flutter/flutter/releases/tag/3.27.1)
+
+- [flutter/160041](https://github.com/flutter/flutter/issues/160041) - [Impeller][Android] Disables Impeller on older Android devices.
+- [flutter/](https://github.com/flutter/flutter/issues/160206) - [Impeller][Android] Disables Android HardwareBuffer based swapchains on all devices.
```

Comment:
is the link on 32 and 33 correct, or should they have the issue number in the text?

Reply:
thanks for the catch, fixed.